### PR TITLE
Release notes draft for 6.0.5

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -5,6 +5,108 @@ It also includes enhancements to the Eventing and Search services.
...
+
+[#table_fixedissues_v605-kv-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-41804[MB-41804^]
+| *Summary*: Fixed an issue where ephemeral buckets set to `auto_delete` with replicas configured could reach a deadlock due to eviction.
+
+| https://issues.couchbase.com/browse/MB-41255[MB-41255^]
+| *Summary*: Fixed an issue where the replication of deletes with XATTRS failed if a EWOULD_BLOCK occurred.
```
Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc]: "EWOULD_BLOCK" doesn't mean much to end-users. Suggest re-phrasing to "... with XATTRs failed if the affected document was non-resident."

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -5,6 +5,108 @@ It also includes enhancements to the Eventing and Search services.
...
+
+| https://issues.couchbase.com/browse/MB-40531[MB-40531^]
+| *Summary*: Fixed an issue where the HiFi-MRU Item Pager evicted items from active vBuckets even when there were ample replica vBucket items which could be evicted first.
```
Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc]: This is essentially a performance regression, so I think re-phrasing something along the lines of:

> Fixed a performance regression where the HiFi-MRU Item Pager (added in 5.0) would evict more active items compared to previous LRU ItemPager under certain workloads.

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -5,6 +5,108 @@ It also includes enhancements to the Eventing and Search services.
...
+
+| https://issues.couchbase.com/browse/MB-38741[MB-38741^]
+| *Summary*: The frequency at which the expiry pager runs can be set using a configuration parameter `exp_pager_stime`. Since the default value of 3600s was seen to impact front-end operations, the default value has been reduced to 600s.
+
+| https://issues.couchbase.com/browse/MB-38283[MB-38283^]
+| *Summary*: Fixed an issue where Memcached log wrapped due to excessive DCP "no memory" messages.
+
+| https://issues.couchbase.com/browse/MB-34949[MB-34949^]
+| *Summary*:  Fixed an issue where StateLock was not held in all cases.
```
Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc]: Can probably expand this a bit - something like:

Fixed a theoretical race condition where if a vBucket changed state away from active at the same time as a Delete operation, the delete could incorrectly report success; but would not be applied to new active vBucket.

**Review by daverigby — CHANGES_REQUESTED** [review]
I also see no mention of https://issues.couchbase.com/browse/MB-41999 (probably should be under cluster manager).

**Comment by arod1987** [comment]
Reviewed everything except GSI/N1QL - looks good to me

**Review by daverigby — CHANGES_REQUESTED** [review]
Data Service looks good; but still need https://issues.couchbase.com/browse/MB-41999 including.

**Review by mihirkamdarqe — APPROVED** [review]

**Comment by amarantha-k** [comment]
MB-41999 is a private ticket and I confirmed with @b33f that it is not to be included yet as the issue is still open in other versions.

**Pull request merged**